frontend/public/components/cron-job.tsx

```diff
@@ -124,57 +83,66 @@ const CronJobTableRow: RowFunction<CronJobKind> = ({ obj: cronjob, index, key, s
 
 const CronJobDetails: React.FC<CronJobDetailsProps> = ({ obj: cronjob }) => {
   const job = cronjob.spec.jobTemplate;
+  const { t } = useTranslation();
   return (
     <>
       <div className="co-m-pane__body">
         <div className="row">
           <div className="col-md-6">
-            <SectionHeading text="CronJob Details" />
+            <SectionHeading text={t('workload~CronJob Details')} />
             <ResourceSummary resource={cronjob}>
-              <DetailsItem label="Schedule" obj={cronjob} path="spec.schedule" />
-              <DetailsItem label="Concurrency Policy" obj={cronjob} path="spec.concurrencyPolicy" />
+              <DetailsItem label={t('workload~Schedule')} obj={cronjob} path="spec.schedule" />
               <DetailsItem
-                label="Starting Deadline Seconds"
+                label={t('workload~Concurrency Policy')}
+                obj={cronjob}
+                path="spec.concurrencyPolicy"
+              />
+              <DetailsItem
+                label={t('workload~Starting Deadline Seconds')}
                 obj={cronjob}
                 path="spec.startingDeadlineSeconds"
               >
                 {cronjob.spec.startingDeadlineSeconds
-                  ? pluralize(cronjob.spec.startingDeadlineSeconds, 'second')
-                  : 'Not Configured'}
+                  ? pluralize(cronjob.spec.startingDeadlineSeconds, t('workload~second'))
```

Comment:
Don't use `pluralize`. You want to use i18next plural messages.

https://www.i18next.com/translation-function/plurals